sdk/sync/network_deprecated.go

```diff
@@ -80,12 +66,16 @@ type LinkShape struct {
 }
 
 // LinkRule applies a LinkShape to a subnet.
+//
+// Deprecated
 type LinkRule struct {
 	LinkShape
 	Subnet net.IPNet
 }
 
 // NetworkConfig specifies how a node's network should be configured.
+//
+// Deprecated
```

Comment:
Where's the replacement?

Reply:
Ah, I see:

> The network-related methods have been marked deprecated in the sync package, and will be moved to a network package in an upcoming refactor.

Please don't deprecate without replacement, that will cause a ton of lint failures and confusion. We can always deprecate these when we actually move them.

Reply:
Yeah, I kinda jumped the gun. I'll remove the deprecations.